company/vision.md

```diff
@@ -0,0 +1,80 @@
+---
+meta:
+    title: Company Vision
+---
+
+### Vision
+
+> We will only be successful if the whole Node-RED community is successful.
+
+Our company vision has four parts and captures what we want FlowForge to evolve into in the next 5 years. The vision is inspiration for the [product strategy](../product/strategy.md), which is updated continuously. It’s based on the four pillars. This allows us to adapt to changing market conditions so that we meet customer needs.
+
+- Drive the growth and success of the [Node-RED](https://nodered.org) project
+- Build a platform that becomes the standard way of running Node-RED at any scale - Be DevOps for Node-RED
+- Drive adoption of Node-RED in manufacturing
+- Build an inclusive, high-performance team
+
+#### Drive the growth and success of the Node-RED project
+
+Node-RED is at the core of what we do. Node-RED enables tens of thousands to create and maintain workflows and logic of both physical devices and digital services. Ensuring their effectiveness grows, while new users are introduced to Node-RED is vital to the success of FlowForge.
+
+FlowForge will drive the Node-RED project through:
+
+- **Deliver features** to the core of Node-RED that are stable and provide value to the community
+- **Actively participate** in the community to establish a positive reputation and level of engagement
+- **Be model citizens** for the open governance of the project. Our activities that impact the core are discussed and developed in the open.
+- **Establish close relationships** with other stakeholders of the project - those who have their own commercial interest in the success of the project. Encourage participation and engagement in the core project.
+- **Driving adoption** of Node-RED through our advocacy and our actions.
+
+#### Become the standard way of running any Node-RED at any scale - Be DevOps for Node-RED
+
+There's a wide range of deployment scenarios and FlowForge aims to build a
+platform to help each of them to be successful. FlowForge will solve challenges
+unsolved by Node-RED and push Low-Code beyond the capabilities of Node-RED.
+
+Firstly, team collaboration. Allowing Flows to be developed by multiple
+individuals, ensuring quality and correctness. Push the boundaries of Low-Code
+to allow more people to collaborate.
+
+Secondly, Node-RED is deployed in many different environments. FlowForge will allow
+everyone to achieve the best results whatever the execution context.
```

Comment:
This point is pretty vague. I think we should be identifying what we think are the challenges. Such as: security, high availability, security.

Reply:
Not sure where you want product strategy vs corporate vision. In point #1 you mention collaboration but that seems to be similar to HA & Scalability.